clients/python-wrapper/lakefs/object.py

```diff
@@ -183,7 +208,6 @@ class ObjectReader(LakeFSIOBase):
     ObjectReader provides read-only functionality for lakeFS objects with IO semantics.
     This Object is instantiated and returned for immutable reference types (Commit, Tag...)
     """
-
```

Comment:
I think that `discard()` on a Reader should throw away the buffer, e.g. like `self._readlines_buf = io.BytesIO()`

Reply:
In that sense - there's no real difference between calling reader.close() or reader.discard(). Do you see a use-case where you'd want to specifically call discard on reader and not simply close()?

Reply:
Ah right, I had a wrong idea of what `BytesIO.close()` does. I think this is fine, then.